package.json

```diff
@@ -14,35 +14,39 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.12.13",
-    "@bcrs-shared-components/enums": "1.0.13",
+    "@bcrs-shared-components/breadcrumb": "2.1.24",
+    "@bcrs-shared-components/corp-type-module": "1.0.13",
+    "@bcrs-shared-components/enums": "1.0.44",
     "@bcrs-shared-components/genesys-web-message": "1.0.0",
-    "@bcrs-shared-components/interfaces": "1.0.22",
+    "@bcrs-shared-components/interfaces": "1.0.67",
     "@bcrs-shared-components/staff-payment": "1.0.29",
     "@bcrs-shared-components/web-chat": "2.0.0",
     "@mdi/font": "^4.5.95",
     "@sentry/browser": "^7.49.0",
     "@sentry/integrations": "^7.49.0",
-    "axios": "^0.21.1",
+    "axios": "^0.27.2",
     "babel-plugin-transform-decorators": "^6.24.1",
-    "core-js": "^3.8.0",
+    "core-js": "^3.30.1",
     "http-status-codes": "^1.4.0",
-    "launchdarkly-js-client-sdk": "^2.18.3",
-    "lodash": "^4.17.20",
+    "launchdarkly-js-client-sdk": "^2.24.2",
+    "lodash": "^4.17.21",
     "remove-accents": "^0.4.2",
     "sbc-common-components": "3.0.3",
     "style-loader": "^1.3.0",
-    "vue": "^2.6.12",
+    "vue": "2.7.10",
```

Comment:
Let's remember to try v2.7.14.

Reply:
If I'm not mistaken, that'll cause build time to be super fast. At least, that was the case in our UIs. Could be different here.

Reply:
Yeah, there was a problem at one time, and then it all went away when ... I can't remember! But the other UIs are at 2.7.14.

Reply:
When we upgraded to Vite 🥳 That's why I was able to upgrade them to 2.7.14.